home_work/hw_11/bank_deposit/currency.py

```diff
@@ -0,0 +1,36 @@
+import requests
+from loguru import logger as log
+
+API_KEY = '1240d98b1bc1385eea7ad602'
+API_URL = f"https://v6.exchangerate-api.com/v6/{API_KEY}/latest/USD"
+
+
+class CurrencyConverter:
+
+    @classmethod
+    def _get_exchange_rates(cls):
+
+        response = requests.get(API_URL, timeout=5)
+        response.raise_for_status()
+        return response.json()['conversion_rates']
+
+    @classmethod
+    def convert(cls,
```

Comment:
```suggestion
    def convert(self,
```

Reply:
Probably it's better to use @staticmethod here? Because this class doesn't have any state(atributes) ?

Reply:
Definitely